mkosi/__init__.py

```diff
@@ -1872,6 +1872,11 @@ def build_uki(
         "--ro-bind", stub, stub,
     ]
 
+    if context.config.devicetree:
+        dtb = context.root / f"usr/lib/linux-image-{kver}" / context.config.devicetree
```

Comment:
There should probably be some checking here for the case that the file does not exist in the image.

Reply:
The "other" options like e.g. initrds also do not check the existence of the files.

If a non existing dtb will be configured the current output looks like this:

```
‣  Generating unified kernel image for kernel version 6.6.15-arm64
bwrap: Can't find source path /home/manut/.cache/mkosi/mkosi-workspaceuw1qmalv/root/usr/lib/linux-image-6.6.15-arm64/idonot/exist.dtb: No such file or directory
bwrap: Can't find source path /home/manut/.cache/mkosi/mkosi-workspaceuw1qmalv/root/usr/lib/linux-image-6.6.15-arm64/idonot/exist.dtb: No such file or directory
‣ "/usr/bin/ukify --cmdline @/home/manut/.cache/mkosi/mkosi-workspaceuw1qmalv/cmdline --os-release @/home/manut/.cache/mkosi/mkosi-workspaceuw1qmalv/root/usr/lib/os-release --stub /home/manut/.cache/mkosi/mkosi-workspaceuw1qmalv/root/usr/lib/systemd/boot/efi/linuxaa64.efi.stub --output /home/manut/.cache/mkosi/mkosi-workspaceuw1qmalv/root/boot/EFI/Linux/debian-6.6.15-arm64.efi --efi-arch aa64 --uname 6.6.15-arm64 --devicetree /home/manut/.cache/mkosi/mkosi-workspaceuw1qmalv/root/usr/lib/linux-image-6.6.15-arm64/idonot/exist.dtb --sign-kernel --signtool sbsign --secureboot-private-key /home/manut/qemu/mkosi.key --secureboot-certificate /home/manut/qemu/mkosi.crt --pcr-private-key /home/manut/qemu/mkosi.key --pcr-banks sha1,sha256 build --linux /home/manut/.cache/mkosi/mkosi-workspaceuw1qmalv/root/usr/lib/modules/6.6.15-arm64/vmlinuz --initrd /home/manut/.cache/mkosi/mkosi-workspaceuw1qmalv/initrd/image.raw-initrd" returned non-zero exit code 1.
```

Shall we fail with a nicer looking message?